pkgs/merkle-tree-wasm/src/merkle_tree_wasm.rs

```diff
@@ -125,12 +121,11 @@ impl<F: PrimeField, const WIDTH: usize> MerkleTree<F, WIDTH> {
 
         let mut current_layer = &self.layers[0];
 
-        let leaf_index_result = self.leaves.iter().position(|&x| x == leaf);
-
-        let mut leaf_index = match leaf_index_result {
-            Some(index) => index,
-            None => return Err("MerkleProof: Leaf is not found, please consider using an address belongs to the anonymous set.".to_string()),
-        };
+        let mut leaf_index = self
+            .leaves
+            .iter()
+            .position(|&x| x == leaf)
+            .context("merkle tree leaf not found")?;
```

Comment:
Did you test this change?

Reply:
added in baedfcd0909c0b13951619ecbfa050a97b6a0e39